fog/ingest/client/src/config.rs

```diff
@@ -23,6 +25,13 @@ fn parse_duration_in_seconds(src: &str) -> Result<Duration, std::num::ParseIntEr
     Ok(Duration::from_secs(u64::from_str(src)?))
 }
 
+fn parse_ristretto_hex(src: &str) -> Result<CompressedRistrettoPublic, String> {
+    let mut key_bytes = [0u8; 32];
+    hex2bin(src.as_bytes(), &mut key_bytes[..])
+        .map_err(|err| format!("Hex decode error: {:?}", err))?;
+    Ok(CompressedRistrettoPublic::from(&key_bytes))
```

Comment:
At minimum, this should use [`hex::decode_to_slice()`](https://docs.rs/hex/0.4.3/hex/fn.decode_to_slice.html), which is already a dep here, instead of adding `hex2bin`.

Best-case scenario is that CompressedRistrettoPublic simply implements [`hex::FromHex`](https://docs.rs/hex/0.4.3/hex/trait.FromHex.html).

Reply:
using hex::decode_to_slice now